src/features/targeting/types.js

```diff
@@ -1,4 +1,37 @@
 // @flow
+import type { SetEligibleMessageHeaders, TargetingApi } from './targeting-api';
+import { messageTypes } from './constants';
+
+export type MessageType = $Values<typeof messageTypes>;
+
+export type MessageID = number;
+
+export type MessageName = string;
+
+type MessageHeaderIdentifiers = {|
+    id: MessageID, // an integer used to load message in getMessageRepresentation
+    name: MessageName, // used to form messageIdentifier where namespace is 'contextual'
+|};
+
+type MessageHeaderParams = {|
+    canBeShownMultipleTimes: boolean, // whether it can be shown again
+    createdDate: number,
+    isCustomMessage: boolean, // if it is custom message, custom message maker must exist to show
```

Comment:
Can the comments be moved to the top of each line? Easier to read and makes it consistent with the above file + allows multi lines.